iban-tools.gemspec

```diff
@@ -2,7 +2,7 @@ Gem::Specification.new do |s|
   s.platform     = Gem::Platform::RUBY
   s.summary      = "IBAN validator"
   s.name         = 'iban-tools'
-  s.version      = '1.0.0'
+  s.version      = '1.2.0'
```

Comment:
@leonardoaretakis we are about to merge another update, can you do `1.2.1`, please?